src/demo/data.js

```diff
@@ -2,51 +2,153 @@ import faker from 'faker/locale/en';
+    filteredLearnerEnrollments = learnerEnrollments.filter(enrollment => enrollment.last_activity_date.getTime() >= lastActivityDate.getTime()); // eslint-disable-line max-len
+  } else if (filterOption === 'inactive_past_week') {
+    lastActivityDate.setDate(lastActivityDate.getDate() - 7);
+    filteredLearnerEnrollments = learnerEnrollments.filter(enrollment => enrollment.last_activity_date.getTime() <= lastActivityDate.getTime()); // eslint-disable-line max-len
+  } else if (filterOption === 'inactive_past_month') {
+    lastActivityDate.setDate(lastActivityDate.getDate() - 30);
+    filteredLearnerEnrollments = learnerEnrollments.filter(enrollment => enrollment.last_activity_date.getTime() <= lastActivityDate.getTime()); // eslint-disable-line max-len
+  }
+  return filteredLearnerEnrollments;
+};
+
+const filterPastWeekCompletions = (pastWeekEnrollments) => {
+  const today = new Date();
+  const passedDateTime = new Date().setDate(today.getDate() - 7);
+  return pastWeekEnrollments.filter(enrollment => enrollment.has_passed &&
+    enrollment.passed_timestamp.getTime() >= passedDateTime &&
+    enrollment.passed_timestamp.getTime() <= today.getTime());
+};
+
+const filterCompletedLearnerCourses = () => enrollments.filter(enrollment => enrollment.has_passed).map((enrollment) => { // eslint-disable-line max-len
+  const totalCurrentEnrollements = enrollments.filter(currentEnrollment => currentEnrollment.user_email === enrollment.user_email).length; // eslint-disable-line max-len
+  return {
+    user_email: enrollment.user_email,
+    completed_courses: totalCurrentEnrollements,
+  };
+});
+
+const filteredEnrollments = (options) => {
+  let filterEnrollments = enrollments;
+  if (options.learner_activity) {
+    filterEnrollments = filterLearnerActivity(filterEnrollments, options.learner_activity);
+  }
+  if (options.passed_date) {
+    filterEnrollments = filterPastWeekCompletions(filterEnrollments);
+  }
+  if (options.has_enrollments && options.extra_fields === 'enrollment_count') {
+    filterEnrollments = filterEnrolledLearners(filterEnrollments);
+  }
+  return filterEnrollments;
+};
+
 const overview = {
-  // We could derive this data from the enrollments but for now just using static data
   active_learners: {
-    past_month: 111,
-    past_week: 48,
+    past_month: filterPastWeekCompletions(enrollments).length,
+    past_week: filterLearnerActivity(enrollments, 'active_past_week').length,
   },
-  course_completions: 85,
-  enrolled_learners: 203,
+  course_completions: filterCompletedLearnerCourses().length,
+  enrolled_learners: filterEnrolledLearners().length,
   last_update_date: new Date(),
-  number_of_users: 268,
+  number_of_users: enrollments.length,
 };
 
+const getEnrollmentsCsv = () => enrollments.reduce((csvData, enrollment) => `${csvData}${Object.values(enrollment).toString()}\n`, `${Object.keys(enrollments[0]).toString()}\n`); // eslint-disable-line max-len
```

Comment:
Can this line be formatted to avoid using `// eslint-disable-line max-len`? It would probably also make it easier to read.